index.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html class="no-js" lang="">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title></title>
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="description" content="">
+
+</head>
+
+<body>
+
+<div class="ticket">
+    <div class="details">
+        <div class="left-section">
+            <h1>CINEMA TICKET</h1>
+            <div class="ticket-type and price">
+                <h2 class="vip">VIP</h2>
+                <h2 class="price">$10</h2>
+            </div>
+        </div>
+        <div class="middle-section">
+            <p>08:00 PM</p>
+            <p>GATE: 03 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; SEAT: B43</p>
```

Comment:
Please don't use the space sign for positioning, instead use CSS rules, for example flexbox